src/main.go

```diff
@@ -1,83 +0,0 @@
-package main
```

Comment:
Main removed, kinda hard to run the application then? :)

Reply:
Fixed :)